target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -434,6 +434,10 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"1:lan:3" "2:lan:4" "3:lan:1" "4:lan:2" "0:wan" "6@eth0"
 		;;
+	tplink,archer-c5-v4)
+		ucidef_add_switch "switch1" \
```

Comment:
"switch1" instead of "switch0" by intention?
(Haven't read the driver stuff ...)

Reply:
`switch0` is mt7620 integrated switch, which is not used here